acapy_plugin_toolbox/credential_definitions.py

```diff
@@ -170,18 +217,20 @@ async def handle(self, context: RequestContext, responder: BaseResponder):
                 attributes=schema["attrNames"],
                 state=SchemaRecord.STATE_WRITTEN,
                 author=SchemaRecord.AUTHOR_OTHER,
+                support_revocation=support_revocation,
```

Comment:
This line shouldn't have been committed. I never modified the base Schema Record, IIRC